Lib/ufo2ft/outlineCompiler.py

```diff
@@ -939,10 +943,19 @@ def setupTable_CFF(self):
         unitsPerEm = round(getAttrWithFallback(info, "unitsPerEm"))
         topDict.FontMatrix = [1.0 / unitsPerEm, 0, 0, 1.0 / unitsPerEm, 0, 0]
         # populate the width values
-        defaultWidthX = round(getAttrWithFallback(info, "postscriptDefaultWidthX"))
+        if not any(hasattr(info, attr) and getattr(info, attr) is not None
+                   for attr in ("postscriptDefaultWidthX",
+                                "postscriptNominalWidthX")):
+            # no custom values set in fontinfo.plist; compute optimal ones
```

Comment:
since the optimal default and nominal widths must be computed at the same time (either both or none of the two), where I check if user has defined any of them in the fontinfo.plist. They usually go in pair, but it's conceivable that only one is defined. In which case I decided to fall back to the old behavior.